mlflow/xgboost.py

```diff
@@ -176,6 +186,8 @@ def log_model(
     signature: ModelSignature = None,
     input_example: ModelInputExample = None,
     await_registration_for=DEFAULT_AWAIT_MAX_SLEEP_SECONDS,
+    pip_requirements=None,
+    additional_pip_requirements=None,
```

Comment:
Can we make some arguments for `log_model` keyword-only to pul related parameters closer?

Reply:
This is a good callout. We can do this in MLflow 2.0. Unfortunately, it would be a breaking change for a pre-2.0 release.